Src/BootCamp.Chapter/GlobalSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public static class GlobalSettings
+    {
+        public static string filePath = Directory.GetCurrentDirectory();
+
+        public static ILogger Logger(LogType logType)
```

Comment:
Methods should be verbs. What do you do with logger?
- get logger?
- create logger?
- load logger?
?

Reply:
Correct, renamed this to LogMessage.